wondrous-app/components/profile/modals/styles.tsx

```diff
@@ -65,7 +88,65 @@ export const UserProfilePicture = (props) => {
 export const DefaultProfilePicture = (props) => {
   return <DefaultUserImage style={ProfilePictureStyles} />;
 };
+export const FlexTitle = styled.div`
```

Comment:
same with the naming, can we make it more informative?

Reply:
done